sources/cloudfront/cache_policy.go

```diff
@@ -65,15 +65,15 @@ func NewCachePolicySource(client CloudFrontClient, accountID string) *sources.Ge
 		},
 		ListFunc: cachePolicyListFunc,
 		ItemMapper: func(_, scope string, awsItem *types.CachePolicy) (*sdp.Item, error) {
-			attributes, err := sources.ToAttributesCase(awsItem)
+			attributes, err := sources.ToAttributesWithExclude(awsItem)
 
 			if err != nil {
 				return nil, err
 			}
 
 			item := sdp.Item{
 				Type:            "cloudfront-cache-policy",
-				UniqueAttribute: "id",
+				UniqueAttribute: "Id",
```

Comment:
For common terms or names should this not be ID ?

Reply:
You'd think that, but the AWS API returns it like this, capital `I` lowercase `d`, so I want to just stick with exactly what the API returns